### ISSUE TITLE
Support distribution Flatcar 

### ISSUE THREAD

**Issue by mazzy89** [kubernetes/kops#7075]
**1. Describe IN DETAIL the feature/behavior/change you would like to see.**

I'd like to include the support for the distribution Flatcar https://www.flatcar-linux.org/ a friendly-fork of CoreOS.

I've already tried to run kops with Flatcar but I get this:

```
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620655    1459 identify.go:91] unhandled os-release info "NAME=\"Flatcar Linux by Kinvolk\"\nID=flatcar\nID_LIKE
=coreos\nVERSION=2079.4.0\nVERSION_ID=2079.4.0\nBUILD_ID=2019-05-15-1954\nPRETTY_NAME=\"Flatcar Linux by Kinvolk 2079.4.0 (Rhyolite)\"\nANSI_COLOR=\"38;5;75\"\nHOME_URL=\"https://flatca
r-linux.org/\"\nBUG_REPORT_URL=\"https://issues.flatcar-linux.org\"\nFLATCAR_BOARD=\"amd64-usr\"\n"
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620722    1459 identify.go:108] unhandled /etc/os-release info "NAME=\"Flatcar Linux by Kinvolk\"\nID=flatcar\nI
D_LIKE=coreos\nVERSION=2079.4.0\nVERSION_ID=2079.4.0\nBUILD_ID=2019-05-15-1954\nPRETTY_NAME=\"Flatcar Linux by Kinvolk 2079.4.0 (Rhyolite)\"\nANSI_COLOR=\"38;5;75\"\nHOME_URL=\"https://
flatcar-linux.org/\"\nBUG_REPORT_URL=\"https://issues.flatcar-linux.org\"\nFLATCAR_BOARD=\"amd64-usr\"\n"
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620741    1459 identify.go:113] could not determine distro
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620750    1459 identify.go:114]   /etc/lsb-release: "DISTRIB_ID=\"Flatcar Linux by Kinvolk\"\nDISTRIB_RELEASE=20
79.4.0\nDISTRIB_CODENAME=\"Rhyolite\"\nDISTRIB_DESCRIPTION=\"Flatcar Linux by Kinvolk 2079.4.0 (Rhyolite)\"\n"
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620761    1459 identify.go:115]   /etc/debian_version: ""
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620768    1459 identify.go:116]   /etc/redhat-release: ""
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620779    1459 identify.go:117]   /usr/lib/os-release: "NAME=\"Flatcar Linux by Kinvolk\"\nID=flatcar\nID_LIKE=$
oreos\nVERSION=2079.4.0\nVERSION_ID=2079.4.0\nBUILD_ID=2019-05-15-1954\nPRETTY_NAME=\"Flatcar Linux by Kinvolk 2079.4.0 (Rhyolite)\"\nANSI_COLOR=\"38;5;75\"\nHOME_URL=\"https://flatcar$linux.org/\"\nBUG_REPORT_URL=\"https://issues.flatcar-linux.org\"\nFLATCAR_BOARD=\"amd64-usr\"\n"
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620794    1459 identify.go:118]   /etc/os-release: "NAME=\"Flatcar Linux by Kinvolk\"\nID=flatcar\nID_LIKE=core$
s\nVERSION=2079.4.0\nVERSION_ID=2079.4.0\nBUILD_ID=2019-05-15-1954\nPRETTY_NAME=\"Flatcar Linux by Kinvolk 2079.4.0 (Rhyolite)\"\nANSI_COLOR=\"38;5;75\"\nHOME_URL=\"https://flatcar-lin$x.org/\"\nBUG_REPORT_URL=\"https://issues.flatcar-linux.org\"\nFLATCAR_BOARD=\"amd64-usr\"\n"
May 27 22:33:48 ip-172-20-33-119.ec2.internal bash[1437]: W0527 22:33:48.620818    1459 main.go:142] got error running nodeup (will retry in 30s): error determining OS distribution: ca$not identify distro
```

**2. Feel free to provide a design supporting your feature request.**

CoreOS is already supported by kops. In order to support Flatcar, I guess I would have to extend the logic here https://github.com/kubernetes/kops/blob/b2d90fd2c0e466a48ff5ebbfca472564f21b21a7/nodeup/pkg/distros/identify.go.

Any other thing I'd have to change?

Thank you

**Comment by mazzy89** [comment]
Closing because implemented and merged

**Comment by jeckhart** [comment]
Fixed by #7084
